download/main.py

```diff
@@ -82,16 +71,9 @@ def main():
             'XRef_Match',
         ]).sample(n=20))
 
-    elections.df.to_csv('.local/elections.csv', index=False)
-    committees.df.to_csv('.local/committees.csv', index=False)
-    a_contributions.df.to_csv('.local/a_contributions.csv', index=False)
-
-    '''
-    with engine.connect() as conn:
-        elections.to_sql(conn)
-        committees.to_sql(conn)
-        a_contributions.to_sql(conn)
-    '''
+    elections.pl.write_csv(f'{OUTPUT_DIR}/elections.csv')
+    committees.pl.write_csv(f'{OUTPUT_DIR}/committees.csv')
+    a_contributions.df.write_csv(f'{OUTPUT_DIR}/a_contributions.csv')
```

Comment:
Is `df` misnamed?  I think you want to replace all the `df` with `pl`?  Or all the `pl` with `df` to be consistent.